src/kernel/arch/x86/arch.zig

```diff
@@ -178,3 +178,10 @@ pub fn haltNoInterrupts() noreturn {
         halt();
     }
 }
+
+///
+/// Register an interrupt handler. The interrupt number should be the arch-specific number.
+///
```

Comment:
Could add the comments from arch_mock to here as this is the more important one

Reply:
Good spot

Reply:
Good spot